docs/versioned-plugins/codecs-index.asciidoc

```diff
@@ -4,20 +4,36 @@
 include::include/plugin-intro.asciidoc[]
 
 include::codecs/avro-index.asciidoc[]
+
```

Comment:
If possible, we should put empty lines between include statements. Several months ago, I ran into a very obscure bug that took more a long time to resolve. I can't remember the details (it had something to do with dynamic IDs that weren't resolving correctly), but the fix was to always include empty lines between includes. If this isn't an easy fix, we can worry about it later if we see issues.